hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStartSubcommand.java

```diff
@@ -38,49 +38,98 @@ public class ContainerBalancerStartSubcommand extends ScmSubcommand {
+          "the cluster after which a datanode will be rebalanced. Value " +
+          "should be in the range [0.0, 100.0) and default value is 10 " +
+          "(for example, '10' for 10%%).")
   private Optional<Double> threshold;
 
   @Option(names = {"-i", "--iterations"},
-      description = "Maximum consecutive iterations that" +
-          " balancer will run for.")
+      description = "Maximum consecutive iterations that " +
+          "balancer will run for. Value should be positive " +
+          "or -1 and default value is 10 (for example, '10' for 10 iterations).")
   private Optional<Integer> iterations;
 
   @Option(names = {"-d", "--max-datanodes-percentage-to-involve-per-iteration",
       "--maxDatanodesPercentageToInvolvePerIteration"},
       description = "Max percentage of healthy, in service datanodes " +
-          "that can be involved in balancing in one iteration (for example, " +
-          "'20' for 20%%).")
+          "that can be involved in balancing in one iteration. Value " +
+          "should be in the range [0,100] and default value is 20 (for " +
+          "example, '20' for 20%%).")
   private Optional<Integer> maxDatanodesPercentageToInvolvePerIteration;
 
   @Option(names = {"-s", "--max-size-to-move-per-iteration-in-gb",
       "--maxSizeToMovePerIterationInGB"},
       description = "Maximum size that can be moved per iteration of " +
-          "balancing (for example, '500' for 500GB).")
+          "balancing. Value should be positive and default value is 500 " +
+          "(for example, '500' for 500GB).")
   private Optional<Long> maxSizeToMovePerIterationInGB;
 
   @Option(names = {"-e", "--max-size-entering-target-in-gb",
       "--maxSizeEnteringTargetInGB"},
       description = "Maximum size that can enter a target datanode while " +
-          "balancing. This is the sum of data from multiple sources (for " +
-          "example, '26' for 26GB).")
+          "balancing. This is the sum of data from multiple sources. Value " +
+           "should be positive and default value is 26 (for example, " +
+          "'26' for 26GB).")
   private Optional<Long> maxSizeEnteringTargetInGB;
 
   @Option(names = {"-l", "--max-size-leaving-source-in-gb",
       "--maxSizeLeavingSourceInGB"},
       description = "Maximum size that can leave a source datanode while " +
-          "balancing. This is the sum of data moving to multiple targets " +
+          "balancing. This is the sum of data moving to multiple targets. " +
+          "Value should be positive and default value is 26 " +
           "(for example, '26' for 26GB).")
   private Optional<Long> maxSizeLeavingSourceInGB;
 
+  @Option(names = {"--balancing-iteration-interval-minutes"},
+      description = "The interval period in minutes between each iteration of Container Balancer. " +
+          "Value should be greater than '0' and default value is 70 (for example, '70' for 70 minutes).")
+  private Optional<Long> balancingInterval;
```

Comment:
Since time is being specified in minutes, we don't need to make this a long. Integer should be enough. Same comment for the other time related variables.